_episodes_rmd/12-supp-factors.Rmd

```diff
@@ -193,7 +193,7 @@ dat$Gender[dat$Gender == 'M'] <- 'm'
 
 > ## Updating Factors
 >
-> ```{r updating-factors}
+> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females.}
```

Comment:
From this error:

```
processing file: _episodes_rmd/12-supp-factors.Rmd
(*) NOTE: I saw chunk options "updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females."
 please go to https://yihui.org/knitr/options
 (it is likely that you forgot to quote "character" options)
Error in parse(text = code, keep.source = FALSE) : 
  <text>:1:38: unexpected INCOMPLETE_STRING
1: alist( 'updating-factors', fig.alt = "Barplot graph showing blood pressure in males and females. )
```

```suggestion
> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females."}
```